docs/azure/sdk/authentication/credential-chains.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Credential chains in the Azure Identity library for .NET'
+description: 'This article describes the DefaultAzureCredential and ChainedTokenCredential classes in the Azure Identity library.'
+ms.topic: conceptual
+ms.date: 08/06/2024
+---
+
+# Credential chains in the Azure Identity library for .NET
+
+The Azure Identity library provides *credentials*&mdash;public classes derived from the Azure Core library's [TokenCredential](/dotnet/api/azure.core.tokencredential?view=azure-dotnet&preserve-view=true) class. A credential represents a distinct authentication flow for acquiring an access token from Microsoft Entra ID. These credentials can be chained together to form an ordered sequence of authentication mechanisms to be attempted.
```

Comment:
nit: the emphasis around `credentials` reads a little strong, maybe we can remove it especially since we are explaining the concept immediately after the dash

Reply:
This type of emphasis is a common practice in docs to call out that we're about to define an important term used throughout the doc.